2021-02-28 12:47:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:47:56 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 12:48:00 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 12:48:00 [INFO] : --------------------------------------------
2021-02-28 12:48:00 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:48:00 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 12:48:01 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 12:48:01 [INFO] : --------------------------------------------
2021-02-28 12:48:01 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:48:01 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 12:48:05 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 12:48:05 [INFO] : --------------------------------------------
2021-02-28 12:48:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:48:05 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 12:48:07 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 12:48:07 [INFO] : --------------------------------------------
2021-02-28 12:48:07 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:48:07 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 12:48:11 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 12:48:11 [INFO] : --------------------------------------------
2021-02-28 12:48:11 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:48:11 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 12:48:22 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 12:48:22 [INFO] : --------------------------------------------
2021-02-28 12:48:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 12:48:22 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 12:48:25 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521]
2021-02-28 12:48:25 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 12:48:25 [INFO] : --------------------------------------------
2021-02-28 13:18:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:18:43 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 13:18:48 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 13:18:48 [INFO] : --------------------------------------------
2021-02-28 13:18:48 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:18:48 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 13:18:49 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 13:18:49 [INFO] : --------------------------------------------
2021-02-28 13:18:49 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:18:49 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 13:18:52 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 13:18:52 [INFO] : --------------------------------------------
2021-02-28 13:18:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:18:52 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 13:18:56 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 13:18:56 [INFO] : --------------------------------------------
2021-02-28 13:18:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:18:56 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 13:18:59 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 13:18:59 [INFO] : --------------------------------------------
2021-02-28 13:18:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:18:59 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 13:19:08 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 13:19:08 [INFO] : --------------------------------------------
2021-02-28 13:19:08 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:19:08 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 13:19:12 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521]
2021-02-28 13:19:12 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 13:19:12 [INFO] : --------------------------------------------
2021-02-28 13:54:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:23 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 13:54:30 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 13:54:30 [INFO] : --------------------------------------------
2021-02-28 13:54:30 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:30 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 13:54:31 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 13:54:31 [INFO] : --------------------------------------------
2021-02-28 13:54:31 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:31 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 13:54:33 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 13:54:33 [INFO] : --------------------------------------------
2021-02-28 13:54:33 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:33 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 13:54:36 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 13:54:36 [INFO] : --------------------------------------------
2021-02-28 13:54:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:36 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 13:54:43 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 13:54:43 [INFO] : --------------------------------------------
2021-02-28 13:54:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:43 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 13:54:51 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 13:54:51 [INFO] : --------------------------------------------
2021-02-28 13:54:51 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 13:54:51 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 13:55:04 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521]
2021-02-28 13:55:04 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 13:55:04 [INFO] : --------------------------------------------
2021-02-28 14:06:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:05 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 14:06:08 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 14:06:08 [INFO] : --------------------------------------------
2021-02-28 14:06:08 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:08 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 14:06:12 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 14:06:12 [INFO] : --------------------------------------------
2021-02-28 14:06:12 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:12 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 14:06:19 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 14:06:19 [INFO] : --------------------------------------------
2021-02-28 14:06:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:19 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 14:06:21 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 14:06:21 [INFO] : --------------------------------------------
2021-02-28 14:06:21 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:21 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 14:06:28 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 14:06:28 [INFO] : --------------------------------------------
2021-02-28 14:06:28 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:28 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 14:06:36 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 14:06:36 [INFO] : --------------------------------------------
2021-02-28 14:06:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:06:36 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 14:06:41 [ERROR] : 获取 [CNVD] 威胁情报异常
Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/cnvd.py", line 63, in get_cves
    log.warn('获取 [%s] 威胁情报失败： [HTTP Error %i] 服务器返回内容：[%s]' % (self.NAME_CH(), response.status_code),response.text)
TypeError: not enough arguments for format string
2021-02-28 14:06:41 [ERROR] : Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/cnvd.py", line 63, in get_cves
    log.warn('获取 [%s] 威胁情报失败： [HTTP Error %i] 服务器返回内容：[%s]' % (self.NAME_CH(), response.status_code),response.text)
TypeError: not enough arguments for format string
Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/cnvd.py", line 63, in get_cves
    log.warn('获取 [%s] 威胁情报失败： [HTTP Error %i] 服务器返回内容：[%s]' % (self.NAME_CH(), response.status_code),response.text)
TypeError: not enough arguments for format string
2021-02-28 14:06:41 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 14:06:41 [INFO] : --------------------------------------------
2021-02-28 14:18:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:18:59 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 14:19:02 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 14:19:02 [INFO] : --------------------------------------------
2021-02-28 14:19:02 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:19:02 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 14:19:04 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 14:19:04 [INFO] : --------------------------------------------
2021-02-28 14:19:04 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:19:04 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 14:19:09 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 14:19:09 [INFO] : --------------------------------------------
2021-02-28 14:19:09 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:19:09 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 14:19:13 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 14:19:13 [INFO] : --------------------------------------------
2021-02-28 14:19:13 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:19:13 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 14:19:23 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 14:19:23 [INFO] : --------------------------------------------
2021-02-28 14:19:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:19:23 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 14:19:37 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 14:19:37 [INFO] : --------------------------------------------
2021-02-28 14:19:37 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:19:37 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 14:19:40 [ERROR] : 获取 [CNVD] 威胁情报异常
Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/cnvd.py", line 63, in get_cves
    log.warn('获取 [%s] 威胁情报失败： [HTTP Error %i] 服务器返回内容：[%s]' % (self.NAME_CH(), response.status_code),response.text)
TypeError: not enough arguments for format string
2021-02-28 14:19:40 [ERROR] : Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/cnvd.py", line 63, in get_cves
    log.warn('获取 [%s] 威胁情报失败： [HTTP Error %i] 服务器返回内容：[%s]' % (self.NAME_CH(), response.status_code),response.text)
TypeError: not enough arguments for format string
Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/cnvd.py", line 63, in get_cves
    log.warn('获取 [%s] 威胁情报失败： [HTTP Error %i] 服务器返回内容：[%s]' % (self.NAME_CH(), response.status_code),response.text)
TypeError: not enough arguments for format string
2021-02-28 14:19:40 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 14:19:40 [INFO] : --------------------------------------------
2021-02-28 14:20:32 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:20:32 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 14:20:34 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 14:20:34 [INFO] : --------------------------------------------
2021-02-28 14:20:34 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:20:34 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 14:20:36 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 14:20:36 [INFO] : --------------------------------------------
2021-02-28 14:20:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:20:36 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 14:20:38 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 14:20:38 [INFO] : --------------------------------------------
2021-02-28 14:20:38 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:20:38 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 14:20:41 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 14:20:41 [INFO] : --------------------------------------------
2021-02-28 14:20:41 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:20:41 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 14:21:43 [ERROR] : 获取 [安全客] 威胁情报异常
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
2021-02-28 14:21:43 [ERROR] : Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
2021-02-28 14:21:43 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 14:21:43 [INFO] : --------------------------------------------
2021-02-28 14:21:43 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:21:43 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 14:21:54 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 14:21:54 [INFO] : --------------------------------------------
2021-02-28 14:21:54 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 14:21:54 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 14:21:57 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~0+'')+([2]*(3)+'')+((+true)+'')+(-~(3)+'')+((1+[0])/[2]+'')+(0+1+0+1+'')+(2+'')+(-~{}+'')+(-~{}+'')+(9-1*2+'')+('.')+(1+5+'')+((1+[2])/[2]+'')+(+!+[]+'')+('|')+('-')+(+!+[]+'')+('|')+('l')+((1<<3)+'')+('T')+('N')+(3+4+'')+('E')+('d')+('D')+('S')+('N')+(8+'')+([2]*(3)+'')+(-~[6]+'')+('%')+(1+1+'')+('B')+('s')+('Y')+('y')+('X')+('Z')+('A')+('%')+(0+1+0+1+'')+('F')+('B')+('d')+('z')+('m')+('J')+('A')+('%')+((2^1)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1|2)+'')+((1+[2])/[2]+'')+(~~''+'')+((+[])+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 14:21:57 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 14:21:57 [INFO] : --------------------------------------------
2021-02-28 15:18:48 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:18:48 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 15:18:50 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 15:18:50 [INFO] : --------------------------------------------
2021-02-28 15:18:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:18:50 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 15:18:51 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 15:18:51 [INFO] : --------------------------------------------
2021-02-28 15:18:51 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:18:51 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 15:18:55 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 15:18:55 [INFO] : --------------------------------------------
2021-02-28 15:18:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:18:55 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 15:18:57 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 15:18:57 [INFO] : --------------------------------------------
2021-02-28 15:18:57 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:18:57 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 15:19:00 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 15:19:00 [INFO] : --------------------------------------------
2021-02-28 15:19:00 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:19:00 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 15:19:08 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 15:19:08 [INFO] : --------------------------------------------
2021-02-28 15:19:08 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 15:19:08 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 15:19:11 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~false+'')+(6+'')+((+true)+'')+((2<<1)+'')+(([2]+0>>2)+'')+(2+'')+(([2]+0>>2)+'')+(-~(4)+'')+(1+4+'')+(-~false+'')+('.')+(~~[]+'')+(-~(8)+'')+(3+6+'')+('|')+('-')+(+!+[]+'')+('|')+(~~false+'')+('S')+('P')+('e')+('x')+('w')+('h')+('N')+('h')+((1+[2]>>2)+'')+('a')+('W')+('h')+('h')+(-~1+'')+('l')+('r')+('X')+('q')+('S')+(4+5+'')+('D')+('v')+('k')+('K')+('H')+('E')+('%')+(1+2+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(3+'')+(1+5+'')+((+false)+'')+(~~''+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 15:19:11 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 15:19:11 [INFO] : --------------------------------------------
2021-02-28 16:31:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:18 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 16:31:19 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 16:31:19 [INFO] : --------------------------------------------
2021-02-28 16:31:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:19 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 16:31:20 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 16:31:20 [INFO] : --------------------------------------------
2021-02-28 16:31:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:20 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 16:31:22 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 16:31:22 [INFO] : --------------------------------------------
2021-02-28 16:31:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:22 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 16:31:28 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 16:31:28 [INFO] : --------------------------------------------
2021-02-28 16:31:28 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:28 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 16:31:36 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 16:31:36 [INFO] : --------------------------------------------
2021-02-28 16:31:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:36 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 16:31:45 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 16:31:45 [INFO] : --------------------------------------------
2021-02-28 16:31:45 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 16:31:45 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 16:31:51 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~{}+'')+((1+[2])/[2]+'')+((+true)+'')+(4+'')+((1+[0])/[2]+'')+(+!+[]*2+'')+(2+7+'')+(1+[0]-(1)+'')+(-~false+'')+(-~0+'')+('.')+(-~[6]+'')+(3+'')+('|')+('-')+(-~false+'')+('|')+('P')+('Y')+(2+3+'')+('m')+('I')+('K')+('a')+('H')+('m')+('G')+('G')+((+true)+'')+('%')+((1<<1)+'')+('F')+(~~[]+'')+('A')+('w')+(9+'')+('o')+('T')+('y')+('q')+('r')+('h')+('R')+('Q')+('m')+('s')+('%')+((1+[2]>>2)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1|2)+'')+(6+'')+((+[])+'')+(~~''+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 16:31:51 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 16:31:51 [INFO] : --------------------------------------------
2021-02-28 17:31:08 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:08 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 17:31:09 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 17:31:09 [INFO] : --------------------------------------------
2021-02-28 17:31:09 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:09 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 17:31:10 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 17:31:10 [INFO] : --------------------------------------------
2021-02-28 17:31:10 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:10 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 17:31:12 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 17:31:12 [INFO] : --------------------------------------------
2021-02-28 17:31:12 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:12 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 17:31:15 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 17:31:15 [INFO] : --------------------------------------------
2021-02-28 17:31:15 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:15 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 17:31:18 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 17:31:18 [INFO] : --------------------------------------------
2021-02-28 17:31:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:18 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 17:31:26 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 17:31:26 [INFO] : --------------------------------------------
2021-02-28 17:31:26 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 17:31:26 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 17:31:28 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~0+'')+((1+[2])/[2]+'')+(-~{}+'')+(1+3+'')+(2+3+'')+((1+[2]>>2)+'')+(3+'')+((2)*[2]+'')+(2+6+'')+((2<<2)+'')+('.')+(1+4+'')+((1+[2])/[2]+'')+(2+2+'')+('|')+('-')+(-~false+'')+('|')+('H')+('M')+('x')+('C')+('t')+('Y')+('A')+(2+'')+((2<<1)+'')+('F')+('O')+('D')+('G')+(1+7+'')+('i')+('V')+('k')+((1+[0])/[2]+'')+('f')+('y')+('P')+('c')+('J')+('X')+('R')+('d')+('Q')+('%')+((1+[2]>>2)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(3+'')+(6+'')+((+false)+'')+(~~[]+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 17:31:28 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 17:31:28 [INFO] : --------------------------------------------
2021-02-28 18:21:03 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:03 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 18:21:05 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 18:21:05 [INFO] : --------------------------------------------
2021-02-28 18:21:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:05 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 18:21:06 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 18:21:06 [INFO] : --------------------------------------------
2021-02-28 18:21:06 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:06 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 18:21:08 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 18:21:08 [INFO] : --------------------------------------------
2021-02-28 18:21:08 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:08 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 18:21:11 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 18:21:11 [INFO] : --------------------------------------------
2021-02-28 18:21:11 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:11 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 18:21:13 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 18:21:13 [INFO] : --------------------------------------------
2021-02-28 18:21:13 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:13 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 18:21:20 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 18:21:20 [INFO] : --------------------------------------------
2021-02-28 18:21:20 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 18:21:20 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 18:21:22 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~0+'')+([2]*(3)+'')+(-~0+'')+((2<<1)+'')+(2+3+'')+(1+2+'')+(1+5+'')+((2)*[2]+'')+(-~[7]+'')+((1<<1)+'')+('.')+(1+6+'')+(-~(3)+'')+(4+5+'')+('|')+('-')+(-~{}+'')+('|')+('S')+('p')+('D')+('z')+('D')+('h')+(-~[]+'')+(-~[]+'')+(1+4+'')+('d')+('S')+(1+3+'')+('q')+('F')+('c')+('c')+(2+2+'')+('W')+('L')+(~~{}+'')+('t')+((1+[2]>>2)+'')+('j')+('b')+('o')+('N')+('M')+('%')+(3+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(-~[2]+'')+(-~[5]+'')+(~~{}+'')+(~~[]+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 18:21:22 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 18:21:22 [INFO] : --------------------------------------------
2021-02-28 19:14:47 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:14:47 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 19:14:49 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 19:14:49 [INFO] : --------------------------------------------
2021-02-28 19:14:49 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:14:49 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 19:14:50 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 19:14:50 [INFO] : --------------------------------------------
2021-02-28 19:14:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:14:50 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 19:14:53 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 19:14:53 [INFO] : --------------------------------------------
2021-02-28 19:14:53 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:14:53 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 19:14:55 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 19:14:55 [INFO] : --------------------------------------------
2021-02-28 19:14:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:14:55 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 19:14:58 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 19:14:58 [INFO] : --------------------------------------------
2021-02-28 19:14:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:14:58 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 19:15:05 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 19:15:05 [INFO] : --------------------------------------------
2021-02-28 19:15:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 19:15:05 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 19:15:08 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(+!+[]+'')+(6+'')+(-~false+'')+((2<<1)+'')+(1+4+'')+(-~[2]+'')+([3]*(3)+'')+(-~[6]+'')+(~~[]+'')+(2+6+'')+('.')+(+!+[]+'')+((1|2)+'')+(2+7+'')+('|')+('-')+(+!+[]+'')+('|')+('N')+('x')+('H')+('L')+('S')+(+!+[]*2+'')+('X')+('%')+(-~1+'')+('F')+('H')+('i')+('o')+('F')+('B')+('Q')+('N')+(2+2+'')+('c')+(-~[2]+'')+('y')+('P')+(-~1+'')+('P')+('f')+('a')+('H')+('i')+(1+7+'')+('%')+(1+2+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(3+'')+(2+4+'')+(~~[]+'')+(~~{}+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 19:15:08 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 19:15:08 [INFO] : --------------------------------------------
2021-02-28 20:19:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:19:52 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 20:19:54 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 20:19:54 [INFO] : --------------------------------------------
2021-02-28 20:19:54 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:19:54 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 20:19:55 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 20:19:55 [INFO] : --------------------------------------------
2021-02-28 20:19:55 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:19:55 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 20:19:56 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 20:19:56 [INFO] : --------------------------------------------
2021-02-28 20:19:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:19:56 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 20:19:59 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 20:19:59 [INFO] : --------------------------------------------
2021-02-28 20:19:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:19:59 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 20:20:02 [INFO] : 得到 [安全客] 最新威胁情报 [2] 条
2021-02-28 20:20:02 [INFO] : --------------------------------------------
2021-02-28 20:20:02 [INFO] : 
==============================================
[ TITLE ] Zenphoto CMS 1.5.7 Shell Upload - CXSecurity.com
[ TIME  ] 2021-02-28 20:20:02
[ CVE   ] CVE-2020-36079
[ SRC   ] 安全客
[ URL   ] https://www.anquanke.com/vul/id/2362804
2021-02-28 20:20:02 [INFO] : 
==============================================
[ TITLE ] VisualWare MyConnection Server 11.x Remote Code Execution - CXSecurity.com
[ TIME  ] 2021-02-28 20:20:02
[ CVE   ] CVE-2021-27198
[ SRC   ] 安全客
[ URL   ] https://www.anquanke.com/vul/id/2362803
2021-02-28 20:20:02 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:20:02 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 20:20:09 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 20:20:09 [INFO] : --------------------------------------------
2021-02-28 20:20:09 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 20:20:09 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 20:20:11 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(+!+[]+'')+([2]*(3)+'')+(-~{}+'')+(-~(3)+'')+(5+'')+((2)*[2]+'')+((1|2)+'')+([2]*(3)+'')+((+true)+'')+(-~0+'')+('.')+(1+3+'')+(4+'')+(-~(4)+'')+('|')+('-')+((+true)+'')+('|')+('O')+('Q')+('E')+('n')+('B')+('N')+('%')+(2+'')+('F')+('H')+('v')+('p')+('p')+(9+'')+(9-1*2+'')+('e')+('l')+(2+6+'')+('c')+('i')+('U')+((+false)+'')+('y')+('Q')+('d')+('d')+(-~[]+'')+('%')+(-~1+'')+('F')+('A')+('%')+(3+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(-~[2]+'')+((1+[2])/[2]+'')+((+[])+'')+((+[])+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 20:20:11 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 20:20:11 [INFO] : --------------------------------------------
2021-02-28 20:20:11 [INFO] : [邮件] 正在通过 Github Actions 推送威胁情报...
2021-02-28 21:16:29 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:29 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 21:16:30 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 21:16:30 [INFO] : --------------------------------------------
2021-02-28 21:16:30 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:30 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 21:16:31 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 21:16:31 [INFO] : --------------------------------------------
2021-02-28 21:16:31 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:31 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 21:16:34 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 21:16:34 [INFO] : --------------------------------------------
2021-02-28 21:16:34 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:34 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 21:16:36 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 21:16:36 [INFO] : --------------------------------------------
2021-02-28 21:16:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:36 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 21:16:39 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 21:16:39 [INFO] : --------------------------------------------
2021-02-28 21:16:39 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:39 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 21:16:46 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 21:16:46 [INFO] : --------------------------------------------
2021-02-28 21:16:46 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 21:16:46 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 21:16:48 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~false+'')+(-~[5]+'')+(-~{}+'')+(4+'')+(-~(4)+'')+((1<<2)+'')+(-~[6]+'')+(~~[]+'')+((+false)+'')+(9-1*2+'')+('.')+(3+5+'')+(-~[7]+'')+(1+5+'')+('|')+('-')+(-~[]+'')+('|')+('V')+('l')+((+[])+'')+('z')+('R')+(7+'')+('t')+('n')+(6+'')+('s')+(-~[]+'')+('y')+('L')+('L')+('l')+('%')+(0+1+0+1+'')+('B')+('t')+('U')+('J')+('P')+((2^1)+'')+('L')+('d')+('s')+('E')+((1<<1)+'')+('U')+('%')+(3+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(3+'')+(3+3+'')+(~~''+'')+(~~{}+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 21:16:48 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 21:16:48 [INFO] : --------------------------------------------
2021-02-28 22:20:29 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:29 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 22:20:30 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 22:20:30 [INFO] : --------------------------------------------
2021-02-28 22:20:30 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:30 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 22:20:31 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 22:20:31 [INFO] : --------------------------------------------
2021-02-28 22:20:31 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:31 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 22:20:36 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 22:20:36 [INFO] : --------------------------------------------
2021-02-28 22:20:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:36 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 22:20:38 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 22:20:38 [INFO] : --------------------------------------------
2021-02-28 22:20:38 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:38 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 22:20:41 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 22:20:41 [INFO] : --------------------------------------------
2021-02-28 22:20:41 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:41 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 22:20:48 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 22:20:48 [INFO] : --------------------------------------------
2021-02-28 22:20:48 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 22:20:48 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 22:20:50 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(+!+[]+'')+((1+[2])/[2]+'')+(-~0+'')+(1+3+'')+(2+3+'')+(5+'')+(~~[]+'')+((2<<2)+'')+(2+3+'')+((+[])+'')+('.')+([2]*(3)+'')+(~~[]+'')+((2<<1)+'')+('|')+('-')+((+true)+'')+('|')+('d')+('v')+('B')+('r')+(1+6+'')+((1|2)+'')+('x')+('P')+('j')+('n')+('s')+('e')+(1+5+'')+('e')+('V')+('v')+('M')+('F')+('Z')+(1+8+'')+(1+1+'')+('f')+('n')+('N')+('Z')+('N')+('Y')+('%')+((2^1)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1|2)+'')+(3+3+'')+(~~{}+'')+(~~''+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 22:20:50 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 22:20:50 [INFO] : --------------------------------------------
2021-02-28 23:20:09 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:09 [INFO] : 正在获取 [360] 威胁情报...
2021-02-28 23:20:10 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-02-28 23:20:10 [INFO] : --------------------------------------------
2021-02-28 23:20:10 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:10 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-02-28 23:20:11 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-02-28 23:20:11 [INFO] : --------------------------------------------
2021-02-28 23:20:11 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:11 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-02-28 23:20:13 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-02-28 23:20:13 [INFO] : --------------------------------------------
2021-02-28 23:20:13 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:13 [INFO] : 正在获取 [红后] 威胁情报...
2021-02-28 23:20:15 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-02-28 23:20:15 [INFO] : --------------------------------------------
2021-02-28 23:20:15 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:15 [INFO] : 正在获取 [安全客] 威胁情报...
2021-02-28 23:20:18 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-02-28 23:20:18 [INFO] : --------------------------------------------
2021-02-28 23:20:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:18 [INFO] : 正在获取 [斗象] 威胁情报...
2021-02-28 23:20:25 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-02-28 23:20:25 [INFO] : --------------------------------------------
2021-02-28 23:20:25 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-02-28 23:20:25 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-02-28 23:20:28 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~false+'')+((1+[2])/[2]+'')+(-~false+'')+(4+'')+(5+'')+(2+3+'')+(-~(3)+'')+(-~(3)+'')+((1<<1)+'')+((1<<3)+'')+('.')+((+[])+'')+((1+[2]>>2)+'')+(1+4+'')+('|')+('-')+(+!+[]+'')+('|')+('E')+('B')+('n')+(4+5+'')+((2)*[4]+'')+('j')+(5+'')+('R')+('m')+('F')+('F')+(~~{}+'')+('f')+('z')+('N')+(1+7+'')+('i')+('S')+('q')+('N')+('K')+('f')+('u')+(1+1+'')+(-~1+'')+(2+7+'')+(8+'')+('%')+(3+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1+[2]>>2)+'')+(3+3+'')+(~~false+'')+((+false)+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-02-28 23:20:28 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-02-28 23:20:28 [INFO] : --------------------------------------------
2021-03-01 00:51:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:51:50 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 00:51:52 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 00:51:52 [INFO] : --------------------------------------------
2021-03-01 00:51:52 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:51:52 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 00:51:53 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 00:51:53 [INFO] : --------------------------------------------
2021-03-01 00:51:53 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:51:53 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 00:51:54 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 00:51:54 [INFO] : --------------------------------------------
2021-03-01 00:51:54 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:51:54 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 00:51:57 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 00:51:57 [INFO] : --------------------------------------------
2021-03-01 00:51:57 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:51:57 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 00:52:58 [ERROR] : 获取 [安全客] 威胁情报异常
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
2021-03-01 00:52:58 [ERROR] : Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
2021-03-01 00:52:58 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 00:52:58 [INFO] : --------------------------------------------
2021-03-01 00:52:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:52:58 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 00:53:05 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 00:53:05 [INFO] : --------------------------------------------
2021-03-01 00:53:05 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 00:53:05 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 00:53:07 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~{}+'')+(2+4+'')+(+!+[]+'')+((2)*[2]+'')+((1+[0])/[2]+'')+(2+3+'')+(4+5+'')+(3+6+'')+(8+'')+(1+6+'')+('.')+((1+[2]>>2)+'')+(1+6+'')+(1+4+'')+('|')+('-')+(-~{}+'')+('|')+('b')+('Y')+('l')+('%')+(-~1+'')+('F')+('x')+(4+'')+('d')+('u')+('r')+(~~false+'')+('B')+('W')+('A')+('k')+('e')+('w')+('l')+('O')+('B')+('j')+('C')+('D')+('h')+('Z')+('b')+('b')+('Q')+('%')+((1+[2]>>2)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((2^1)+'')+(3+3+'')+((+false)+'')+((+[])+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 00:53:07 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 00:53:07 [INFO] : --------------------------------------------
2021-03-01 01:51:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:22 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 01:51:24 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 01:51:24 [INFO] : --------------------------------------------
2021-03-01 01:51:24 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:24 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 01:51:25 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 01:51:25 [INFO] : --------------------------------------------
2021-03-01 01:51:25 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:25 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 01:51:29 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 01:51:29 [INFO] : --------------------------------------------
2021-03-01 01:51:29 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:29 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 01:51:32 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 01:51:32 [INFO] : --------------------------------------------
2021-03-01 01:51:32 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:32 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 01:51:35 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 01:51:35 [INFO] : --------------------------------------------
2021-03-01 01:51:35 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:35 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 01:51:42 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 01:51:42 [INFO] : --------------------------------------------
2021-03-01 01:51:42 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 01:51:42 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 01:51:45 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~{}+'')+(1+5+'')+((+true)+'')+((2)*[2]+'')+(-~(4)+'')+(2+4+'')+(1+2+'')+(5+'')+((+false)+'')+((2)*[2]+'')+('.')+(9+'')+((2<<2)+'')+((2^1)+'')+('|')+('-')+(-~{}+'')+('|')+(3+4+'')+('s')+('o')+('L')+('t')+('N')+('M')+(1+1+'')+('p')+('q')+('q')+('%')+(0+1+0+1+'')+('B')+(-~(4)+'')+(([2]+0>>2)+'')+('u')+('E')+('R')+(6+'')+('t')+(-~[2]+'')+('u')+('H')+(1+6+'')+('%')+(-~1+'')+('F')+('T')+('H')+('o')+('%')+((2^1)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1|2)+'')+(1+5+'')+(~~false+'')+(~~''+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 01:51:45 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 01:51:45 [INFO] : --------------------------------------------
2021-03-01 02:31:59 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:31:59 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 02:32:01 [INFO] : 得到 [360] 最新威胁情报 [1] 条
2021-03-01 02:32:01 [INFO] : --------------------------------------------
2021-03-01 02:32:01 [INFO] : 
==============================================
[ TITLE ] 安全事件周报 (2.22-2.28)
[ TIME  ] 1970-01-01 00:00:00
[ CVE   ] 
[ SRC   ] 360
[ URL   ] https://cert.360.cn/warning/detail?id=f0f6332808d47a29ed4e2899d872953a
2021-03-01 02:32:01 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:32:01 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 02:32:02 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 02:32:02 [INFO] : --------------------------------------------
2021-03-01 02:32:02 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:32:02 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 02:32:04 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 02:32:04 [INFO] : --------------------------------------------
2021-03-01 02:32:04 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:32:04 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 02:32:06 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 02:32:06 [INFO] : --------------------------------------------
2021-03-01 02:32:06 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:32:06 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 02:32:09 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 02:32:09 [INFO] : --------------------------------------------
2021-03-01 02:32:09 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:32:09 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 02:32:17 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 02:32:17 [INFO] : --------------------------------------------
2021-03-01 02:32:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 02:32:17 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 02:32:19 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+((+true)+'')+((1+[2])/[2]+'')+(+!+[]+'')+(-~(3)+'')+(1+4+'')+(2+4+'')+(1+4+'')+(2+7+'')+(3+'')+(4+5+'')+('.')+((1+[4]>>1)+'')+(5+'')+(1+3+'')+('|')+('-')+(-~[]+'')+('|')+(-~[2]+'')+('y')+('Q')+((+true)+'')+('N')+('X')+(-~0+'')+('K')+(3+6+'')+(-~[7]+'')+('k')+('m')+('T')+('V')+('q')+('Y')+('t')+(4+4+'')+('q')+('X')+('S')+('o')+('X')+('d')+(1+4+'')+('R')+('Q')+('%')+(-~[2]+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(-~[2]+'')+(1+5+'')+((+[])+'')+((+[])+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 02:32:19 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 02:32:19 [INFO] : --------------------------------------------
2021-03-01 02:32:19 [INFO] : [邮件] 正在通过 Github Actions 推送威胁情报...
2021-03-01 03:20:49 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:20:49 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 03:20:50 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 03:20:50 [INFO] : --------------------------------------------
2021-03-01 03:20:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:20:50 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 03:20:51 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 03:20:51 [INFO] : --------------------------------------------
2021-03-01 03:20:51 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:20:51 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 03:20:54 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 03:20:54 [INFO] : --------------------------------------------
2021-03-01 03:20:54 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:20:54 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 03:20:56 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 03:20:56 [INFO] : --------------------------------------------
2021-03-01 03:20:56 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:20:56 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 03:20:58 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 03:20:58 [INFO] : --------------------------------------------
2021-03-01 03:20:58 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:20:58 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 03:21:06 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 03:21:06 [INFO] : --------------------------------------------
2021-03-01 03:21:06 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 03:21:06 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 03:21:08 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~false+'')+(-~[5]+'')+(-~0+'')+((2<<1)+'')+(1+4+'')+(1+5+'')+(-~[7]+'')+(1+7+'')+((1+[2])/[2]+'')+(-~[7]+'')+('.')+((+true)+'')+((+true)+'')+(2+7+'')+('|')+('-')+(+!+[]+'')+('|')+(0+1+0+1+'')+('L')+('e')+('t')+('d')+('E')+('m')+((+[])+'')+('N')+('%')+(+!+[]*2+'')+('F')+('N')+('j')+('d')+('x')+('L')+('s')+('K')+('Q')+('j')+('%')+(0+1+0+1+'')+('B')+('Q')+('c')+('W')+('K')+('N')+((1<<2)+'')+('Y')+('%')+((1+[2]>>2)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1+[2]>>2)+'')+(-~[5]+'')+(~~[]+'')+(~~[]+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 03:21:08 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 03:21:08 [INFO] : --------------------------------------------
2021-03-01 04:24:33 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:33 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 04:24:36 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 04:24:36 [INFO] : --------------------------------------------
2021-03-01 04:24:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:36 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 04:24:37 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 04:24:37 [INFO] : --------------------------------------------
2021-03-01 04:24:37 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:37 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 04:24:39 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 04:24:39 [INFO] : --------------------------------------------
2021-03-01 04:24:39 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:39 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 04:24:41 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 04:24:41 [INFO] : --------------------------------------------
2021-03-01 04:24:41 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:41 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 04:24:44 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 04:24:44 [INFO] : --------------------------------------------
2021-03-01 04:24:44 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:44 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 04:24:53 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 04:24:53 [INFO] : --------------------------------------------
2021-03-01 04:24:53 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 04:24:53 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 04:24:54 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+((+true)+'')+(2+4+'')+(-~{}+'')+(-~(3)+'')+(1+4+'')+(1+6+'')+((1<<1)+'')+(1+5+'')+(1+8+'')+((1<<2)+'')+('.')+(-~[5]+'')+(2+7+'')+(-~false+'')+('|')+('-')+(-~0+'')+('|')+('k')+('i')+('v')+('b')+('S')+('A')+('D')+('d')+('R')+('D')+('C')+('S')+(1+[0]-(1)+'')+('o')+('Z')+('x')+('e')+('r')+(6+'')+('U')+('K')+('M')+('B')+('W')+((2^1)+'')+(+!+[]*2+'')+('w')+('%')+((1+[2]>>2)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1+[2]>>2)+'')+([2]*(3)+'')+(~~''+'')+((+[])+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 04:24:54 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 04:24:54 [INFO] : --------------------------------------------
2021-03-01 05:19:14 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:14 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 05:19:16 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 05:19:16 [INFO] : --------------------------------------------
2021-03-01 05:19:16 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:16 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 05:19:17 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 05:19:17 [INFO] : --------------------------------------------
2021-03-01 05:19:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:17 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 05:19:19 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 05:19:19 [INFO] : --------------------------------------------
2021-03-01 05:19:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:19 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 05:19:22 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 05:19:22 [INFO] : --------------------------------------------
2021-03-01 05:19:22 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:22 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 05:19:25 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 05:19:25 [INFO] : --------------------------------------------
2021-03-01 05:19:25 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:25 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 05:19:33 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 05:19:33 [INFO] : --------------------------------------------
2021-03-01 05:19:33 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 05:19:33 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 05:19:35 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(-~false+'')+(-~[5]+'')+(-~[]+'')+(4+'')+(-~(4)+'')+(7+'')+(-~(4)+'')+(-~(8)+'')+(-~[6]+'')+(1+4+'')+('.')+(-~[2]+'')+(-~1+'')+(5+'')+('|')+('-')+(-~false+'')+('|')+('K')+('a')+('K')+('l')+('f')+('i')+((2)*[2]+'')+('y')+('w')+('Q')+('g')+('v')+('L')+('B')+('%')+((1<<1)+'')+('F')+('b')+('r')+('n')+('j')+('E')+('D')+(-~false+'')+((1+[2]>>2)+'')+('R')+('v')+('D')+((2)*[4]+'')+('%')+(1+2+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1|2)+'')+(-~[5]+'')+(~~''+'')+(~~[]+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 05:19:35 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 05:19:35 [INFO] : --------------------------------------------
2021-03-01 06:22:10 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:10 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 06:22:13 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 06:22:13 [INFO] : --------------------------------------------
2021-03-01 06:22:13 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:13 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 06:22:17 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 06:22:17 [INFO] : --------------------------------------------
2021-03-01 06:22:17 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:17 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 06:22:21 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 06:22:21 [INFO] : --------------------------------------------
2021-03-01 06:22:21 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:21 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 06:22:23 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 06:22:23 [INFO] : --------------------------------------------
2021-03-01 06:22:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:23 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 06:22:31 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 06:22:31 [INFO] : --------------------------------------------
2021-03-01 06:22:31 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:31 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 06:22:39 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 06:22:39 [INFO] : --------------------------------------------
2021-03-01 06:22:39 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 06:22:39 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 06:22:42 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+((+true)+'')+(3+3+'')+(-~[]+'')+((2)*[2]+'')+(([2]+0>>2)+'')+(9-1*2+'')+(-~(8)+'')+(9-1*2+'')+(3+3+'')+(-~1+'')+('.')+(0+1+0+1+'')+((2<<1)+'')+(-~false+'')+('|')+('-')+(-~{}+'')+('|')+('t')+('a')+(9+'')+('C')+('e')+('Z')+('v')+('M')+((+[])+'')+(-~[5]+'')+('S')+('p')+('B')+('P')+('Z')+('R')+('T')+('b')+('S')+('B')+('B')+('q')+('B')+('L')+('N')+(0+1+0+1+'')+('w')+('%')+(-~[2]+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(1+2+'')+(2+4+'')+(~~''+'')+(~~{}+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 06:22:42 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 06:22:42 [INFO] : --------------------------------------------
2021-03-01 07:19:14 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:19:14 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 07:19:18 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 07:19:18 [INFO] : --------------------------------------------
2021-03-01 07:19:18 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:19:18 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 07:19:19 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 07:19:19 [INFO] : --------------------------------------------
2021-03-01 07:19:19 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:19:19 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 07:19:23 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 07:19:23 [INFO] : --------------------------------------------
2021-03-01 07:19:23 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:19:23 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 07:19:25 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 07:19:25 [INFO] : --------------------------------------------
2021-03-01 07:19:25 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:19:25 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 07:20:26 [ERROR] : 获取 [安全客] 威胁情报异常
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
2021-03-01 07:20:26 [ERROR] : Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
    conn.connect()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connection.py", line 362, in connect
    self.sock = ssl_wrap_socket(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/ssl_.py", line 386, in ssl_wrap_socket
    return context.wrap_socket(sock, server_hostname=server_hostname)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 500, in wrap_socket
    return self.sslsocket_class._create(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1040, in _create
    self.do_handshake()
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
socket.timeout: _ssl.c:1108: The handshake operation timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
    retries = retries.increment(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/util/retry.py", line 410, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/packages/six.py", line 735, in reraise
    raise value
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
    httplib_response = self._make_request(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 384, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=conn.timeout)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/urllib3/connectionpool.py", line 335, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/_base_crawler.py", line 64, in cves
    new_cves = self.get_cves()
  File "/home/runner/work/threat-broadcast/threat-broadcast/src/crawler/anquanke.py", line 41, in get_cves
    response = requests.get(
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/api.py", line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
    resp = self.send(prep, **send_kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
    r = adapter.send(request, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/requests/adapters.py", line 529, in send
    raise ReadTimeout(e, request=request)
requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='www.anquanke.com', port=443): Read timed out. (read timeout=60)
2021-03-01 07:20:26 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 07:20:26 [INFO] : --------------------------------------------
2021-03-01 07:20:26 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:20:26 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 07:20:42 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 07:20:42 [INFO] : --------------------------------------------
2021-03-01 07:20:42 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 07:20:42 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 07:20:47 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+(+!+[]+'')+(3+3+'')+(-~0+'')+(1+3+'')+(-~(4)+'')+(3+5+'')+(-~[2]+'')+(1+1+'')+(-~(3)+'')+(1+6+'')+('.')+(2+2+'')+((1|2)+'')+(3+4+'')+('|')+('-')+(-~{}+'')+('|')+('N')+(2+5+'')+('y')+(2+7+'')+('A')+('s')+('g')+('T')+('x')+('l')+('X')+(5+'')+(3+4+'')+(2+7+'')+(5+'')+('k')+('e')+(3+4+'')+('r')+('Y')+('M')+('R')+('O')+('L')+('v')+('r')+('k')+('%')+((1|2)+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+(-~[2]+'')+(6+'')+(~~''+'')+((+[])+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 07:20:47 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 07:20:47 [INFO] : --------------------------------------------
2021-03-01 08:24:29 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:29 [INFO] : 正在获取 [360] 威胁情报...
2021-03-01 08:24:31 [INFO] : 得到 [360] 最新威胁情报 [0] 条
2021-03-01 08:24:31 [INFO] : --------------------------------------------
2021-03-01 08:24:31 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:31 [INFO] : 正在获取 [绿盟] 威胁情报...
2021-03-01 08:24:32 [INFO] : 得到 [绿盟] 最新威胁情报 [0] 条
2021-03-01 08:24:32 [INFO] : --------------------------------------------
2021-03-01 08:24:32 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:32 [INFO] : 正在获取 [奇安信] 威胁情报...
2021-03-01 08:24:36 [INFO] : 得到 [奇安信] 最新威胁情报 [0] 条
2021-03-01 08:24:36 [INFO] : --------------------------------------------
2021-03-01 08:24:36 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:36 [INFO] : 正在获取 [红后] 威胁情报...
2021-03-01 08:24:39 [INFO] : 得到 [红后] 最新威胁情报 [0] 条
2021-03-01 08:24:39 [INFO] : --------------------------------------------
2021-03-01 08:24:39 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:39 [INFO] : 正在获取 [安全客] 威胁情报...
2021-03-01 08:24:42 [INFO] : 得到 [安全客] 最新威胁情报 [0] 条
2021-03-01 08:24:42 [INFO] : --------------------------------------------
2021-03-01 08:24:42 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:42 [INFO] : 正在获取 [斗象] 威胁情报...
2021-03-01 08:24:50 [INFO] : 得到 [斗象] 最新威胁情报 [0] 条
2021-03-01 08:24:50 [INFO] : --------------------------------------------
2021-03-01 08:24:50 [INFO] : ++++++++++++++++++++++++++++++++++++++++++++
2021-03-01 08:24:50 [INFO] : 正在获取 [CNVD] 威胁情报...
2021-03-01 08:24:52 [WARNING] : 获取 [CNVD] 威胁情报失败： [HTTP Error 521] 服务器返回内容：[<script>document.cookie=('_')+('_')+('j')+('s')+('l')+('_')+('c')+('l')+('e')+('a')+('r')+('a')+('n')+('c')+('e')+('_')+('s')+('=')+((+true)+'')+(6+'')+(-~false+'')+(2+2+'')+(2+3+'')+((2<<2)+'')+(2+5+'')+((+[])+'')+(2+7+'')+(1+1+'')+('.')+(~~''+'')+(9+'')+(4+5+'')+('|')+('-')+(-~[]+'')+('|')+('S')+('G')+('P')+('Q')+('l')+('b')+('i')+(3+5+'')+('j')+('g')+('o')+('V')+('v')+('f')+('A')+('p')+(7+'')+('H')+('t')+('B')+(9-1*2+'')+('g')+('m')+(+!+[]+'')+('M')+(0+1+0+1+'')+('k')+('%')+(1+2+'')+('D')+(';')+('m')+('a')+('x')+('-')+('a')+('g')+('e')+('=')+((1+[2]>>2)+'')+(1+5+'')+((+false)+'')+(~~[]+'')+(';')+('p')+('a')+('t')+('h')+('=')+('/');location.href=location.pathname+location.search</script>]
2021-03-01 08:24:52 [INFO] : 得到 [CNVD] 最新威胁情报 [0] 条
2021-03-01 08:24:52 [INFO] : --------------------------------------------
